system-design/orchestration/workload/workload-orchestration-edge-onboarding.md

```diff
@@ -1,62 +1,70 @@
-# Workload Management Interface Onboarding
-In order for the Workload Orchestration Software to manage the Edge Device's workloads, the Workload Management Interface must first be onboarded.
+# Device Management Client Onboarding
+In order for the workload orchestration solution to manage the edge device's workloads, the device's management client must first complete onboarding.
 
-**Onboarding process includes:**
+> Action: The details in this page are still under discussion and have not been finalized.
 
-1. Margo Interface receives WOS API Endpoint and Signing Certificate HTTP Endpoint
-2. Context and Trust is established between Interface and the WOS API Endpoints
-3. Margo Interface receives Git release URL endpoint and interface associated access token
-3. Trust establishment between the interface and the Device's Associated Git Repository 
-4. Device Capability information transfer from Device to Orchestration Software
+**The onboarding process includes:**
```

Comment:
It would be good if the bullet number could be depicted in the steps of the sequence diagram. Maybe we could consider to use Mermaid for versioning diagrams. It is a markdown dialect for all kinds of software engineering diagrams. [Infos](https://github.blog/developer-skills/github/include-diagrams-markdown-files-mermaid/)

Reply:
If there is general interest, I would be willing to transfer the diagrams.

Reply:
Good suggestion, I will take a note of this to ensure I don't lose track of the comment. Will resolve converstation for now as we are looking to get the PR through. Improvements can be done post merge.